great_expectations/expectations/metrics/map_metric.py

```diff
@@ -872,20 +895,15 @@ def _sqlalchemy_map_condition_unexpected_count_value(
             )
         ).scalar()
     except OperationalError as oe:
-        exception_message: str = "An SQL execution Exception occurred.  "
-        exception_traceback: str = traceback.format_exc()
-        exception_message += (
-            f'{type(oe).__name__}: "{str(oe)}".  Traceback: "{exception_traceback}".'
-        )
-        logger.error(exception_message)
```

Comment:
Isn't this info still useful? Or is it too noisy or something?

Reply:
This messes up CLI stdout checks that must be "clean".